provider/service.py

```diff
@@ -145,17 +152,48 @@ def stopChangesFeed(self):
     def createAndRunConsumer(self, doc):
         triggerFQN = doc['_id']
 
+        if doc['isMessageHub']:
+            retries = 0
+            max_retries = 5
+            url = '{}/admin/topics'.format(doc['kafka_admin_url'])
+            username = doc['username']
+            password = doc['password']
+            topic = doc['topic']
+
+            while retries < max_retries:
```

Comment:
Seems like we could have a test here for invalid auth.

Reply:
yeah i'm afraid i'm going to have to change a doc in cloudant for that. need to get that figured out